python/cudf/cudf/_lib/cudf.pyx

```diff
@@ -109,11 +112,11 @@ cpdef get_ctype_ptr(obj):
 
 
 cpdef get_column_data_ptr(obj):
-    return get_ctype_ptr(obj._data.mem)
+    return get_ctype_ptr(obj._data_view())
```

Comment:
Do we not have a more efficient way to get the pointer to the data than to go through Numba?

Reply:
We do. This is just a result of me mindlessly replacing every instance of `obj.data.mem` with `_data_view()`

Reply:
Done